user_modules/shared_testcases/testCasesForRAI.lua

```diff
@@ -0,0 +1,169 @@
+local testCasesForRAI = {}
+
+require('atf.util')
+local events = require("events")
+
+--[[@InitHMI_onReady_without_UI_GetCapabilities: replace original InitHMIOnReady from connecttest 
+--! without expect UI.GetCapabilites
+--! @parameters: NO
+--]]
+function testCasesForRAI:InitHMI_onReady_without_UI_GetCapabilities(self)
```

Comment:
@aderiabin, REJECTED.
In case I don't declare function in this way, script has run-time error:
"PANIC: unprotected error in call to Lua API (./user_modules/shared_testcases/testCasesForRAI.lua:128: attempt to index field 'hmiConnection' (a nil value))"

Please note that this function is based on original one of connecttest.lua and the differences are related only with UI_GetCapabilities.